package.json

```diff
@@ -24,6 +24,8 @@
   },
   "homepage": "https://github.com/beakerbrowser/dat-ephemeral-ext-msg#readme",
   "dependencies": {
+    "hypercore": "^6.24.0",
+    "hyperdb": "github:aral/hyperdb#local-options",
```

Comment:
Looks good except this -- does it need to be on your branch? Also can it be a dev dep?

Reply:
Oh gosh, sorry, forgot about that. Will fix that now. Give me a few minutes to go upstairs :)

Reply:
Right, so that broke the tests! :roll_eyes: 

I think it might be an incompatibility between the version of hypercore/hypercore protocol used by hyperdb and hyperdrive when included in the same app. I had the tests break before and nuking the node_modules directory and `npm i` got them working again then. But it’s clearly fragile. I’m going to look into that now. Will update once I have something that works reliably across various versions.

Where would be a good place to raise the issue of cross-package compatibility? It would make sense that hyperdb and hyperdrive both used the same version of hypercore and the hypercore protocol.